src/app/containers/PageHandlers/withContexts.jsx

```diff
@@ -27,7 +27,11 @@ const WithContexts = Component => {
 
     return (
       <ToggleContextProvider service={service} origin={bbcOrigin}>
-        <ServiceContextProvider service={service} variant={variant}>
+        <ServiceContextProvider
+          service={service}
+          variant={variant}
+          pathname={pathname}
```

Comment:
Should we be worried about ServiceContext loading every time pathname is updated?

Reply:
This is going to re-render currently when `pathname` changes anyway, as the top level `App` component will be re-rendering when pathname changes as things stand.